### PR TITLE
[ISSUE #242] update port

### LINKED ISSUE #242: Inconsistent http port
**BUG REPORT**

1. Please describe the issue you observed:

- What did you do (The steps to reproduce)?

I notice the port of source code is 8081 but in the config file is 8082

>  
    code:
    /**
     * Http port for REST API.
     */
    private int httpPort = 8081;

    config file:
    ## Http port for user to access REST API
    httpPort=8082

If the user is debugging with source code, launching the Connector according to ReadME will report an error

- What is expected to see?

The port of source code is consistent with config file

- What did you see instead?

it is inconsistent

2. Please tell us about your environment:

master

4. Other information (e.g. detailed explanation, logs, related issues, suggestions on how to fix, etc):

### PR DESCRIPTION
**Pull request by Oliverwqcwrw**
## What is the purpose of the change

Close #242 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq-connect/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by odbozhou — APPROVED** [review]

File: rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/config/ConnectConfig.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ public void setBrokerSuspendMaxTimeMillis(int brokerSuspendMaxTimeMillis) {
     /**
      * Http port for REST API.
      */
-    private int httpPort = 8081;
+    private int httpPort = 8082;
```
Comment (ShannonDing) [inline rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/config/ConnectConfig.java:116]: why change to 8082?

Comment (Oliverwqcwrw) [reply rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/config/ConnectConfig.java:116]: I notice the port above README and config file is 8082

Comment (Oliverwqcwrw) [reply rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/config/ConnectConfig.java:116]: I notice the port above README and config file is 8082

**Review by RockteMQ-AI — APPROVED** [review]
LGTM. Trivial default port change (8081 → 8082) to avoid conflict with other services.

---
*Automated review by github-manager*

**Pull request merged**